docs/build/building-apps/04-vote-extensions.md

```diff
@@ -13,7 +13,7 @@ defined in ABCI++.
 
 ABCI++ allows an application to extend a pre-commit vote with arbitrary data. This
 process does NOT have to be deterministic, and the data returned can be unique to the
-validator process. The Cosmos SDK defines `baseapp.ExtendVoteHandler`:
+validator process. The Cosmos SDK defines [`baseapp.ExtendVoteHandler`](https://github.com/cosmos/cosmos-sdk/blob/main/types/abci.go#L26-L27):
```

Comment:
nit: I would avoid using `main` since the code is always subject to change. Instead, point to a static release such as v0.50.0